src/db/query.js

```diff
@@ -63,7 +64,10 @@ async function query(query, db = null, removeNull = false, source = 'modal') {
 
           for (let j = 0; j < e.columns.length; j++) {
             const column = e.columns[j]
-
+            let fieldData = res[j]
+            if (PLATFORM === "WINDOWS" && typeof fieldData === "string") {
```

Comment:
I think this is the place where a comment would be justified explaining why this code is here. Something like `// Single quotes become duplicated on Windows which leads to parsing errors.` WDYT?

Reply:
Can we change this to single quotes `if (PLATFORM === 'WINDOWS' && typeof fieldData === 'string') {`